forum/qa-plugin/user-activity-log/user-activity-log-overrides.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+function qa_admin_sub_navigation(){
+    $nav = qa_admin_sub_navigation_base();
+    if(qa_get_logged_in_level() >= QA_USER_LEVEL_EDITOR){
+        $nav['user-activity-log'] = array(
+            'label' => qa_lang_html('user-activity-log/linkLabel'),
+            'url' => qa_path('user-activity-log'),
+            'selected' => (qa_request_part(1) == 'user-activity-log')? true : null, 
```

Comment:
Nie lepiej użyć tutaj potrójnego operatora porównania zamiast podwójnego? Oraz, czy wartość `null` jest tu odpowiednia, zamiast `false`? Jeśli pole `selected` oczekuje typu boolean, to można pozbyć się ternarki i zapisać po prostu `qa_request_part(1) === 'user-activity-log'`.